.ci/seed_es_on_travis.sh

```diff
@@ -0,0 +1,152 @@
+ES5PLUS_ARCHIVE=https://artifacts.elastic.co/downloads/elasticsearch
+
+# where is the test data? what file has the mapping for the test "shakespeare" index?
+TEST_DATA_DIR=$(pwd)/test_data
+LEGACY_MAPPING_FILE="${TEST_DATA_DIR}/legacy_shakespeare_mapping.json"
+ES5_MAPPING_FILE="${TEST_DATA_DIR}/es5_shakespeare_mapping.json"
+ES6_MAPPING_FILE="${TEST_DATA_DIR}/es6_shakespeare_mapping.json"
+
+
+case "$ES_VERSION" in
+    "") ;;
+
+    "1.0.0")
+      export ES_VERSION=1.0.0;
+      export MAPPING_FILE=${LEGACY_MAPPING_FILE};
+      export ES_BINARY_URL="${ES1_ARCHIVE}/elasticsearch-$ES_VERSION.deb"
+      ;;
+
+    "1.4.4")
+      export ES_VERSION=1.4.4;
+      export MAPPING_FILE=${LEGACY_MAPPING_FILE};
+      export ES_BINARY_URL="${ES1_ARCHIVE}/elasticsearch-$ES_VERSION.deb"
+      ;;
+
+    "1.7.2")
+      export ES_VERSION=1.7.2;
+      export MAPPING_FILE=${LEGACY_MAPPING_FILE};
+      export ES_BINARY_URL="${ES1_ARCHIVE}/elasticsearch-$ES_VERSION.deb"
+      ;;
+
+    "2.0.2")
+      export ES_VERSION=2.0.2;
+      export MAPPING_FILE=${LEGACY_MAPPING_FILE};
+      export ES_BINARY_URL="${ES2_ARCHIVE}/$ES_VERSION/elasticsearch-$ES_VERSION.deb"
+      ;;
+
+    "2.1.2")
+      export ES_VERSION=2.1.2;
+      export MAPPING_FILE=${LEGACY_MAPPING_FILE};
+      export ES_BINARY_URL="${ES2_ARCHIVE}/$ES_VERSION/elasticsearch-$ES_VERSION.deb"
+      ;;
+
+    "2.2.2")
+      export ES_VERSION=2.2.2;
+      export MAPPING_FILE=${LEGACY_MAPPING_FILE};
+      export ES_BINARY_URL="${ES2_ARCHIVE}/$ES_VERSION/elasticsearch-$ES_VERSION.deb"
+      ;;
+
+    "2.3.5")
+      export ES_VERSION=2.3.5;
+      export MAPPING_FILE=${LEGACY_MAPPING_FILE};
+      export ES_BINARY_URL="${ES2_ARCHIVE}/$ES_VERSION/elasticsearch-$ES_VERSION.deb"
+      ;;
+
+    "5.0.2")
+      export ES_VERSION=5.0.2;
+      export MAPPING_FILE=${ES5_MAPPING_FILE};
+      export ES_BINARY_URL="${ES5PLUS_ARCHIVE}/elasticsearch-$ES_VERSION.deb"
+      ;;
+
+    "5.3.3")
+      export ES_VERSION=5.3.3;
+      export MAPPING_FILE=${ES5_MAPPING_FILE};
+      export ES_BINARY_URL="${ES5PLUS_ARCHIVE}/elasticsearch-$ES_VERSION.deb"
+      ;;
+    
+    "5.4.3")
+      export ES_VERSION=5.4.3;
+      export MAPPING_FILE=${ES5_MAPPING_FILE};
+      export ES_BINARY_URL="${ES5PLUS_ARCHIVE}/elasticsearch-$ES_VERSION.deb"
+      ;;
+
+    "5.6.9")
+      export ES_VERSION=5.6.9;
+      export MAPPING_FILE=${ES5_MAPPING_FILE};
+      export ES_BINARY_URL="${ES5PLUS_ARCHIVE}/elasticsearch-$ES_VERSION.deb"
+      ;;
+    
+    "6.0.1")
+      export ES_VERSION=6.0.1;
+      export MAPPING_FILE=${ES6_MAPPING_FILE};
+      export ES_BINARY_URL="${ES5PLUS_ARCHIVE}/elasticsearch-$ES_VERSION.deb"
+      ;;
+    
+    "6.1.4")
+      export ES_VERSION=6.1.4;
+      export MAPPING_FILE=${ES5_MAPPING_FILE};
+      export ES_BINARY_URL="${ES5PLUS_ARCHIVE}/elasticsearch-$ES_VERSION.deb"
+      ;;
+      
+    "6.2.4")
+      export ES_VERSION=6.2.4;
+      export MAPPING_FILE=${ES5_MAPPING_FILE};
```

Comment:
ah here's the error (probably?), I used the wrong mapping file.

Python tests didn't fail because there isn't a `get_fields()` in the Python package yet. Created #164 to document it.